tests/manifest/test_services.py

```diff
@@ -0,0 +1,43 @@
+import os
+import pathlib
+import pytest
+
+from modules.common.YAMLReader import YAMLReader
+from manifest.services import ManifestService, ManifestStatus
+
+
+@pytest.fixture()
+def manifest_config():
+    """Create the test config"""
+    # setup
+    yaml = YAMLReader(None).read_yaml()
+    config = yaml.config.manifest
+    config.output_dir = "tests/resources"
+    config.gcp_bucket = None
+    yield config
+    # teardown
+    manifest_json = pathlib.Path(config.output_dir).joinpath(
+        pathlib.Path(config.file_name)
+    )
+    if manifest_json.exists():
+        os.remove(manifest_json)
+
+
+class TestManifestService:
+    """Test manifest service class"""
+
+    def test_update_manifest_after_run(self, manifest_config) -> None:
+        ms = ManifestService(manifest_config)
```

Comment:
This is a quick way of getting a hold on to a ManifestService instance, although I would look into its factory method, *get_manifest_service()*, as it may be ok with the prepared configuration object, with the added benefit that we keep the ManifestService singleton instantiation code in a single place, abstracted from clients, which gives us freedom for future refactoring